service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/model/common/ExtendedAuthorizationRequest.java

```diff
@@ -60,13 +63,16 @@
 
     private List<Integer> va;
 
-    protected BaseKlarnaRequest(final PayoneConfig config, final String requestType,
-                                final String financingtype, final PaymentWithCartLike paymentWithCartLike) {
-        super(config, requestType, ClearingType.PAYONE_KLV.getPayoneCode());
-
-        this.financingtype = financingtype;
-
-        mapKlarnaPropertiesFromPaymentCart(paymentWithCartLike);
+    protected ExtendedAuthorizationRequest(final PayoneConfig config,
```

Comment:
Maybe it makes sense to add @nullable and @Nonnull annotations to clearly distinguish between params which could be null and those that can't.

Reply:
ef466b188